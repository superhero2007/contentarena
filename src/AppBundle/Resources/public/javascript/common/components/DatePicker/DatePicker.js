import React from 'react';
import PropTypes from 'prop-types';
import DatePicker from 'react-datepicker';

const CaDatePicker = (props) => {
    return (<DatePicker {...props} className={`ca-date-picker ${props.className}`} utcOffset={0}/>);
};

CaDatePicker.defaultProps = {
    className: '',
    showYearDropdown: true,
    scrollableYearDropdown: true,
    yearDropdownItemNumber: 50,
    fixedHeight: true
};

CaDatePicker.propTypes = {
    className: PropTypes.string
};

export default CaDatePicker;