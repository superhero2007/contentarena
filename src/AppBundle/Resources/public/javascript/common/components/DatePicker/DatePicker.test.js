import React from "react";
import renderer from "react-test-renderer";
import DatePicker from "./DatePicker";

it("DatePicker renders correctly", () => {
	const tree = renderer
		.create(testingUtils.withContext(<DatePicker />))
		.toJSON();
	expect(tree).toMatchSnapshot();
});
