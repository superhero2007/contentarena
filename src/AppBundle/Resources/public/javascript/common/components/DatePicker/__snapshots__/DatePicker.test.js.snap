// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker renders correctly 1`] = `
<div>
  <div
    className="react-datepicker-wrapper"
  >
    <div
      className="react-datepicker__input-container"
    >
      <input
        className="ca-date-picker "
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;
